index.html

```diff
@@ -1355,6 +1367,199 @@ <h4>Exchange Examples</h4>
+      <p class="note" title="Multiple interaction initiation variations are possible">
+While the example above is a [=holder=]-initiated interaction, [=issuers=] and
+[=verifiers=] are also capable of initiating an interaction. Similarly, while
+the transmission of the interaction URL here is performed via a QR Code
+(optical medium), an interaction URL can also be transmitted over NFC or
+Bluetooth (wireless medium).
+      </p>
+
+      <section>
+        <h4>Interaction URL Format</h4>
+        <p>
+The format of the <dfn>interaction URL</dfn> MUST conform to the syntax for the
+[[[URL]]] and contain an `iuv` query parameter encoding the interaction URL
+version number, which MUST be `1`. The interaction URL SHOULD be an HTTPS URL
+that contains a transaction-specific interaction identifier. The URL SHOULD be
+opaque and require no URL syntax processing before it is fetched by the
+receiving system. An example of such a URL is shown below:
+        </p>
+
+        <pre class="example nohighlight" title="An interaction URL">
+https://app.example/interactions/z8n38Dp7a?iuv=1
+        </pre>
+      </section>
+
+      <section>
+        <h4>Interaction QR Code Format</h4>
+        <p>
+An <dfn>interaction QR Code</dfn> MUST be an [=interaction URL=]
+expressed as a QR code according to [[[ISO18004]]]. To ensure broad
+interoperability, the length of the [=interaction URL=] SHOULD be as short as
+possible, SHOULD NOT exceed 400 alphanumeric characters, and MUST NOT exceed
+4,296 alphanumeric characters. An example of an interaction QR code can be found
+below:
+        </p>
+
+        <figure id="roles">
+          <img style="margin: auto; width: 33%; display: block;"
+         src="./diagrams/interaction-qr-code.png"
+         alt="A square image filled with black and white dots that is encoding
+a interaction URL">
+          <figcaption style="text-align: center;">
+An interaction QR code for `https://app.example/interactions/z8n38Dp7a?iuv=1`
+          </figcaption>
+        </figure>
+
+      </section>
+
+      <section>
+        <h4>Interaction Protocols Response</h4>
+        <p>
+Performing a retrieval of the [=interaction URL=] results in instructions
+on how to start an interaction with the remote system.
+        </p>
+
+        <p>
+When the [=interaction URL=] is fetched using an `Accept` header of
+`application/json`, a single JSON object containing a `protocols` [=map=] MUST
+be returned where each [=map/key=] is a protocol identifier and each
+[=map/value=] is a URL that can be used to initiate the interaction. For
+example, performing an HTTP GET on the
+`https://app.example/interactions/z8n38Dp7a?iuv=1` [=interaction URL=] might
+result in the following response:
+        </p>
+
+        <pre class="example nohighlight" title="A list of protocols supported for a given interaction">
+{
+  "protocols": {
+    "website": "https://app.example/redirects/z8j3kfk2lQ",
+    "vc-api": "https://app.example/workflows/123/exchanges/987",
```

Comment:
This might be just "vcapi" in existing implementations, we should check those and match what they are doing.

Reply:
Looks likethe code is currently `vcapi`, `OID4VCI`, and `OID4VP`:

https://github.com/digitalbazaar/bedrock-vc-delivery/blob/b3aa0d178d5c12bfcb633fee8e062c19705efe1a/lib/http.js#L146

We should probably make it all lowercase in the spec and make matching case-insensitive in the code.

Reply:
"Fixed" for now in f980030fdfc2704ee848b1ac3e2a01e57449466f.